gcp_pilot/healthcare.py

```diff
@@ -0,0 +1,389 @@
+# More Information: https://cloud.google.com/healthcare-api/docs/reference/rest
+import abc
+from typing import Generator, Dict, Any
+
+from gcp_pilot import exceptions
+from gcp_pilot.base import GoogleCloudPilotAPI, DiscoveryMixin, ResourceType
+
+
+class HealthcareBase(DiscoveryMixin, GoogleCloudPilotAPI, abc.ABC):
+    _scopes = ["https://www.googleapis.com/auth/cloud-healthcare"]
+
+    def __init__(self, **kwargs):
+        super().__init__(
+            serviceName="healthcare",
+            version="v1beta1",
+            cache_discovery=False,
+            **kwargs,
+        )
+
+    def _set_location(self, location: str = None) -> str:
+        # Only some limited regions are available: https://cloud.google.com/healthcare-api/docs/concepts/region
+        return "us"
```

Comment:
Wouldn't it be better to `return location` and set the default value to `"us"`?

Reply:
Sure, I can set the default only.